addons/account/static/tests/reconciliation_tests.js

```diff
@@ -1857,5 +1858,65 @@ QUnit.module('account', {
 
         clientAction.destroy();
     });
+    
+    QUnit.test('Reconcile temporarily and ask to check', function(assert){
+        assert.expect(4);
+        this.params.options.context['to_check'] = true;
+        var clientAction = new ReconciliationClientAction.StatementAction(null, this.params.options);
+
+        testUtils.mock.addMockEnvironment(clientAction, {
+            data: this.params.data,
+            session: {
+                currencies: {
+                    3: {
+                        digits: [69, 2],
+                        position: "before",
+                        symbol: "$"
+                    }
+                }
+            },
+        });
+        clientAction.appendTo($('#qunit-fixture'));
+        var widget = clientAction.widgets[0];
+
+        // Add a line as proposition
+        // open the first line
+        widget.$('.accounting_view tfoot td.cell_label').click()
+
+        var $reconcileForm = widget.$(".create");
+        $reconcileForm.find('.create_account_id input').val('499001 Suspense Account').keydown().keyup()
+        $reconcileForm.find('.create_account_id input').click();
+        $('.ui-autocomplete .ui-menu-item a:contains(499001 Suspense Account)')
+            .trigger('mouseover')
+            .trigger('click');
+        
+        assert.equal($("button.o_validate.btn.btn-secondary.text-warning:first").length, 0, "should not display reconcile button in orange");
+        $reconcileForm.find('.create_to_check input').click()
+        assert.equal($("button.o_validate.btn.btn-secondary.text-warning:first").length, 1, "should display reconcile button in orange");
+
+        testUtils.mock.intercept(clientAction, 'call_service', function (event) {
+            assert.deepEqual(event.data.args[1].args,
+                [[5],[{partner_id: 8,
+                        counterpart_aml_dicts: [],
+                        payment_aml_ids: [],
+                        new_aml_dicts: [{account_id: 499,
+                            credit: 1175,
+                            debit: 0,
+                            analytic_tag_ids: [[6, null, []]],
+                            name: 'SAJ/2014/002 and SAJ/2014/003',
+                        }],
+                        to_check: true,
+                }]],
+                "Should call process_bank_statement_line with to_check set to true");
+            var def = $.Deferred();
+            def.abort = function () {};
+            event.data.callback(def);
+        });
+
+        widget.$("button.o_reconcile.btn.btn-primary:first").click();
+        assert.ok(true, "No error in reconciliation");
+
+        clientAction.destroy();
+    });
```

Comment:
@csnauwaert can you check this test please?

Reply:
seems correct to me